docs/main/kafka-knative-broker/README.md

```diff
@@ -0,0 +1,102 @@
+# Using Knative kafka broker
+If you want to use a Knative broker for communication between the different componenets (Data Index, Job Service and Workflows), you should use a reliable broker, i.e: not in-memory.
+
+Kafka perfectly fullfills this reliability need.
+
+## Pre-requisites
+
+1. A Kafka cluster running, see https://strimzi.io/quickstarts/ for a quickstart setup
+
+## Installation steps
+
+1. Configure and enable kafka broker feature in Knative: https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/
+```console
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-controller.yaml
+oc apply --filename https://github.com/knative-extensions/eventing-kafka-broker/releases/download/knative-v1.14.5/eventing-kafka-broker.yaml
+```
+> [!NOTE]
+> This examples uses`knative-v1.14.5`, this may change, please refer to [the official documentation link](https://knative.dev/docs/eventing/brokers/broker-types/kafka-broker/)
+ * Review the `Security Context Constraints` (`scc`) to be granted to the `knative-kafka-broker-data-plane` service account used by the `kafka-broker-receiver`  deployment:
+```console
+oc get deployments.apps -n knative-eventing kafka-broker-receiver -oyaml | oc adm policy scc-subject-review --filename -
+```
+  * i.e:
+```console
+oc -n knative-eventing adm policy add-scc-to-user nonroot-v2 -z knative-kafka-broker-data-plane
+```
+
+* Make sure the `replication.factor` of your Kafka cluster match the one of the `kafka-broker-config` ConfigMap. With the Strimzi quickstart example, this value is set to `1`:
+```console
+oc patch cm kafka-broker-config -n knative-eventing \
+   --type merge \
+   -p '
+   {
+     "data": {
+       "default.topic.replication.factor": "1"
+     }
+   }'
+```
+  * Wait for the `kafka-broker-receiver` resource to be ready:
+```console
+oc wait --for condition=ready=true pod -l app=kafka-broker-receiver -n knative-eventing --timeout=60s
+```
+
+2. Create kafka broker (Knative `sink`): see https://docs.openshift.com/serverless/1.33/eventing/brokers/kafka-broker.html for more details:
+```Console
+echo "apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  annotations:
+    # case-sensitive
+    eventing.knative.dev/broker.class: Kafka
+  name: kafka-broker
+spec:
+  # Configuration specific to this broker.
+  config:
+    apiVersion: v1
+    kind: ConfigMap
+    name: kafka-broker-config
+    namespace: knative-eventing" | oc apply -n sonataflow-infra -f -
+```
+3. Configure the `sonataflowplatforms.sonataflow.org`: given that the `Orchestrator` is named `orchestrator-sample` and was created under the `orchestrator` namespace:
+```console
+oc -n orchestrator patch orchestrators.rhdh.redhat.com orchestrator-sample --type merge \
+   -p '
+{
+  "spec": {
+    "orchestrator": {
+      "sonataflowPlatform": {
+        "eventing": {
+          "broker": {
+            "name": "<BROKER NAME>",
```

Comment:
the names for the kafka broker are hard-coded above. perhaps they should be change to a place holder or env-var instead mixing  both options?